VideoLocker/src/main/java/org/edx/mobile/base/BaseFragmentActivity.java

```diff
@@ -1,5 +1,7 @@
 package org.edx.mobile.base;
 
+import android.app.AlertDialog;
```

Comment:
should be `android.support.v7.app.AlertDialog` so that we get the new Material style on all API levels.